Doc/library/asyncio-future.rst

```diff
@@ -28,6 +28,9 @@ Future Functions
 
 .. function:: ensure_future(obj, \*, loop=None)
 
+   Schedule the execution of a :ref:`coroutine object <coroutine>`: wrap it in
```

Comment:
The use of the ``:term:`` Sphinx role links to Python's glossary directly. ``:ref:`` would not be incorrect in this case, but ``:term:`` is more specific. The glossary would be probably be the best place to direct users to if they are unfamiliar with coroutines.

Additionally, the extra ``:`` character after the Sphinx role isn't needed. 
```suggestion
   Schedule the execution of a :term:`coroutine object <coroutine>` and wrap it in
```

After adding this change, it may be necessary to split this into an additional line. [reST specifies](https://devguide.python.org/documenting/#use-of-whitespace) that the maximum line width should be 80 characters, this change would place the total for this line at 82.

Reply:
Thanks! I have signed the CLA, updated the title, and made the changes that you requested.